pkgs/by-name/kc/kcl/package.nix

```diff
@@ -55,21 +53,19 @@ buildGoModule rec {
     done
   '';
 
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+    set -o pipefail
+    $out/bin/kcl --version | grep $version
+    $out/bin/kcl <(echo 'hello = "KCL"') | grep "hello: KCL"
+    runHook postInstallCheck
+  '';
+
   # By default, libs and bins are stripped. KCL will crash on darwin if they are.
   dontStrip = stdenv.hostPlatform.isDarwin;
 
-  # env vars https://github.com/kcl-lang/kcl-go/blob/main/pkg/env/env.go#L29
-  postFixup = ''
-    wrapProgram $out/bin/kcl \
-      --prefix PATH : "${
-        lib.makeBinPath [
-          kclvm
-          kclvm_cli
-        ]
-      }" \
-      --prefix KCL_LIB_HOME : "${lib.makeLibraryPath [ kclvm ]}" \
-      --prefix KCL_GO_DISABLE_INSTALL_ARTIFACT : false
-  '';
+  doCheck = !(with stdenv; isDarwin && isAarch64); # turning off tests in this case as ofBorg endlessly waits for a builder
```

Comment:
Are the tests broken or are we just asking this because ofborg is nursing resources?
I'm not sure I like this if it's only because ofBorg since we can just not wait for this build instead

Reply:
I don't like it either.
The tests work and are not broken. For practical purposes they are not runnable on ofBorg as the build waits for ever to start the tests.
This was my attempt to speed up the build and not wait for ofBorg :)

What's the current policy/practice for not running a test on ofBorg for a given architecture?

Are you also saying that we don't need to wait for the tests phase to terminate instead?

Reply:
I'm saying that I would rather merge without waiting for ofBorg than disable tests for all aarch64-darwin since we might miss failures then

Reply:
I'm saying that I would rather merge without waiting for ofBorg than disable tests for all aarch64-darwin since we might miss real failures then

Reply:
Oh great. I'll change it.

I was confused as I don't think I can merge it.

Do I have the access level so that I could merge?

Reply:
I can merge it when this is fixed 👍

Reply:
Ok I've made the change. Thank you.